tools/benchmark/binsize/src/main.rs

```diff
@@ -0,0 +1,27 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::fs;
+use std::string::String;
+
+fn wasm_filesize(dir: &str) -> Result<u64, std::io::Error> {
```

Comment:
Thought: All this tool seems to do is to list a directory and print out the file sizes.  Why not use a pre-existing command line tool for that, like `ls` or `du` ?

Reply:
It's doing a little more, not just printing out the file size but formatting it in a particular way (ndjson in the case) to make GHA Continuous Benchmark happy.
I also would like to let it stand as Rust script as my first exercise in Rust coding, in setting up a cargo target, etc.